flask_common/client.py

```diff
@@ -43,7 +43,8 @@ def open(self, *args, **kwargs):
         api_key = kwargs.pop('api_key', self.api_key)
         if 'headers' not in kwargs:
             kwargs['headers'] = self.get_headers(api_key)
-        return super(ApiClient, self).open(*args, **kwargs)
+        with current_app.app_context():
+            return super(ApiClient, self).open(*args, **kwargs)
```

Comment:
Does it ever make sense to reuse the same app context? If not, should we move it to `Client.open`?